js/bubbling-and-capturing.md

```diff
@@ -0,0 +1,143 @@
+The interactivity of our HTML web page is handled by Javascript. This interactivity is nothing but a bunch of events that the HTML elements undergo. How well these events are handled decide how user-friendly the web page is.
+
+Event Bubbling and Event Capturing are two phases of event propagation/flow in Javascript. Event flow is basically the order in which the events are received on a web page. In Javascript, event flow takes place in three phases -
+
+1. Capture Phase
+2. Target Phase
+3. Bubble Phase
+
+This propagation is bidirectional, from the window to the target and back. What differentiates these phases is the type of listeners that are called.
+
+Let's start by understanding Bubbling first.
+
+## Event Bubbling:
+
+** Bubbling is the flow of events where, when an event takes place on an element, it first runs the handler on itself, then on its parent and then on all of its ancestors. **
```

Comment:
Double star is not needed here. Don't start a paragraph with bold text. Use bold when you want reader to specifically pay attention to a certain thing. The reader already has your attention here.